docs/semgrep-app/sso.md

```diff
@@ -23,6 +23,10 @@ import MoreHelp from "/src/components/MoreHelp"
 
 **SSO (single sign-on)** is a **Team/Enterprise tier feature**. Semgrep supports [OpenID Connect / Oauth2](#openid-connect--oauth2) and [SAML 2.0](#saml-20).
 
+:::info
+Add users through your Single Sign On provider without any additional steps in the Semgrep App after you configure the SSO for your organization. The only required steps to ensure that users are added to Semgrep App are on the side of the SSO provider. There are ••no•• additional emails with links. When you follow the SSO steps, you are able to add onboard users with one click. 
```

Comment:
```suggestion
Add users through your Single Sign On provider without any additional steps in Semgrep App after you configure the SSO for your organization. The only required steps to ensure that users are added to Semgrep App are on the side of the SSO provider. There are **no** additional emails with links. When you follow the SSO steps, you are able to add onboard users with one click. 
```

Not sure what those dots are, since markdown uses asterisks.

Reply:
I am not sure how I was able to create them also... 😅 Thanks @s-santillan! Ah, I see - Jonathan made it to test new pipeline - https://github.com/returntocorp/semgrep-docs/pull/840/commits/ce04d25a0dcce979b6d666abdd038787d6480623